content/swift/concepts/sets/terms/insert.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.insert()'
+Description: 'Add an element at a specified index.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`insert()`** method inserts the given element in the set if it is not already present.
```

Comment:
```suggestion
The **`insert()`** method adds or inserts a given element in the set if it is not already present.
```

Reply:
Can we explain that because a set contains unique items, that is the reason an element won't be added if it's already present? Please put it in your own words and in a format that you think works best